Collatz.java

```diff
@@ -1,6 +1,31 @@
 // Demonstrates the Collatz conjecture.
 public class Collatz {
 	public static void main(String args[]) {
-	    // Replace this comment with your code
+		int numOfSequences = Integer.parseInt(args[0]);
+		boolean isVerbose= args[1].equals("v");
+		for (int i = 0; i < numOfSequences; i++) {
+			int startingNum = i + 1;
+			int counter = 1;
+			if (isVerbose) {
```

Comment:
There are multiple (isVerbose), conditions, I am pretty sure you could have merged some of them with better planning..